### PR TITLE
Upgrade frontend packages

### PR DESCRIPTION
**Pull request by auloin**

### CODE REVIEW

**Review by rdehuyss — CHANGES_REQUESTED** [review]
Great work. I love the `StatCard` and think we should use  a similar approach for the problems and the different state components.
I also have the impression that the overall font-size is smaller. Is that possible or is my eyesight getting worse 🤓?

I also noticed that if there are 2 warnings and an error in the dashboard and I dismiss only one warning, the 2 of them are gone. I also think the error should go on top.

I would also like to add eslint and prettier. However, since trying it, Intellij consumes all my CPU

**Comment by auloin** [comment]
> I also have the impression that the overall font-size is smaller. Is that possible or is my eyesight getting worse 🤓?

I checked with chrome font-sizes are still the same.

> I also noticed that if there are 2 warnings and an error in the dashboard and I dismiss only one warning, the 2 of them are gone. I also think the error should go on top.
> 
Should be fixed.

File: core/src/main/resources/org/jobrunr/dashboard/frontend/src/components/jobs/job-view.js [hunk 1]
```diff
@@ -217,12 +215,18 @@ const JobView = (props) => {
...
+                                        color="inherit"
+                                        onClick={changeSortOrder}
+                                        style={classes.sortButton}
+                                        size="large"><SortDescending/></IconButton>
+                                        : <IconButton
+                                        id="jobhistory-sort-asc-btn"
+                                        color="inherit"
+                                        onClick={changeSortOrder}
+                                        style={classes.sortButton}
+                                        size="large"><SortAscending/></IconButton>
```
[the remark this answers is not in the document]
Comment (auloin) [reply core/src/main/resources/org/jobrunr/dashboard/frontend/src/components/jobs/job-view.js:229]: I simplified it without creating a new component

File: core/src/main/resources/org/jobrunr/dashboard/frontend/src/components/jobs/job-view.js [hunk 2]
```diff
@@ -148,7 +146,7 @@ const JobView = (props) => {
         const mustGoBack = 'delete' === apiStatus.type;
         setApiStatus(null);
         if (mustGoBack) {
-            history.goBack();
+            navigate(-1);
```
Comment (auloin) [inline core/src/main/resources/org/jobrunr/dashboard/frontend/src/components/jobs/job-view.js:149]: Is this still necessary?

**Review by rdehuyss — APPROVED** [review]

**Pull request merged**